packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -314,12 +438,8 @@ const PersonalDetailsForm = ({
                     {'phone' in values && (
                         <FormInputField
                             name='phone'
-                            label={
-                                is_svg || is_appstore || is_mf ? localize('Phone number*') : localize('Phone number')
-                            }
-                            placeholder={
-                                is_svg || is_appstore || is_mf ? localize('Phone number*') : localize('Phone number')
-                            }
+                            label={is_svg || is_mf ? localize('Phone number*') : localize('Phone number')}
```

Comment:
all clients will follow either `is_svg || is_mf` of this condition.right?

Reply:
I guess :)
this line of code I didn’t touch, only removed 'is_appstore'